### PR TITLE
Opensearch getting started, upgrading, and plugin list

### PR DESCRIPTION
**Pull request by lornajane**
Adding more documentation for opensearch now we have a bit more info about how the upgrade process is going to work. Oh and updating the icon!

### CODE REVIEW

**Comment by lornajane** [comment]
Fixes #108. Fixes #109.

File: docs/products/opensearch/concepts/opensearch_backups.rst [hunk 1]
```diff
@@ -17,7 +17,10 @@ Depending on the service plan, we offer single backups for disaster recovery or
 To allow returning to an earlier point in time, Aiven for OpenSearch uses two kinds of backups - hourly and daily. Each type has its own
 backup frequency interval and retention period. Currently, you cannot configure these settings.
 
-Aiven for OpenSearch uses the database forking functionality to create a new OpenSearch service that is cloned from the backup that you select.
+Restore from backup
```
Comment (diawarr) [inline docs/products/opensearch/concepts/opensearch_backups.rst:20]: This kind of duplicates the content in opensearch/howto/restore_opensearch_backup - I'd suggest removing one or the other

File: docs/products/opensearch/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Get Started with Aiven for OpenSearch
+=====================================
+
+Aiven for OpenSearch is available in the `Aiven console <https://console.aiven.io>`_.
+
+Choose the OpenSearch version, your cloud provider and location to deploy to, then choose which plan to use.
+
+Finally, give the service a name and then select "Create Service", and your shiny new OpenSearch database will start building. While it does that, you can already visit the service overview page to see the details of the service.
```
Comment (diawarr) [inline docs/products/opensearch/getting-started.rst:8]: "Create Service" could be bolded to keep consistency for the UI references

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
+Upgrade to OpenSearch
+=====================
+
+For current Aiven for Elasticsearch customers, upgrade is recommended as soon as possible so that we can continue to support your database requirements. Depending on your needs, the various options for testing and upgrading the service are detailed here. The upgrade does not require downtime, and any Kibana service alongside Elasticsearch will be upgraded to OpenSearch Dashboards at the same time.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:4]: upgrade >> upgrading (after the comma in the first sentence)
OR:
upgrade is recommended >> we recommend that you upgrade

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+.. warning::
+    We strongly recommend that you avoid upgrading any library dependencies as part of this upgrade process. Keep your current dependencies at their current versions.
+
+There are three approaches that you can use to upgrade an existing service:
+
+* Create a `fork :doc:/docs/platform/concepts/database-forking` of Aiven for OpenSearch from your existing Aiven for Elasticsearch service. This is a good way to get a fast preview of how Aiven for OpenSearch will work for you.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:11]: Link is not rendering properly, should be >> :doc:`fork </docs/platform/concepts/database-forking>`

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+* Create a fork of Aiven for Elasticsearch from your existing Aiven for Elasticsearch service. This gives you the opportunity to test the process of upgrading an existing Elasticsearch service to OpenSearch without experimenting on your live database.
+* Upgrade from Aiven for Elasticsearch to Aiven for OpenSearch to move either a test or production database to the upgraded state.
+
+The steps for each of these options are detailed in this article.
+
+Fork an existing Elasticsearch to OpenSearch
+--------------------------------------------
+
+1. In the service overview of the web console, choose the "New database fork" button to create a new service from a backup.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:20]: "New database fork" could be bolded to keep consistency for the UI references

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+2. When forking Aiven for Elasticsearch services, there in an additional option to specify the "Service type" as either Elasticsearch or OpenSearch. Choose OpenSearch and give your new service a name.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:21]: "Service type" could be bolded to keep consistency for the UI references

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Use the new OpenSearch service to test your application against before performing an upgrade. We recommend upgrading any dependencies at the same time.
+
+Fork an existing Elasticsearch to Elasticsearch
+-----------------------------------------------
+
+The database fork acts as a clone of your existing Elasticsearch service, but allows you to test that an upgrade will run smoothly.
+
+1. In the service overview of the web console, choose the "New database fork" button to create a new service from a backup.
+2. Check that the "Service type" field is set to Elasticsearch and give your new service a name.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:31]: "Service type" could be bolded to keep consistency for the UI references

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+2. When forking Aiven for Elasticsearch services, there in an additional option to specify the "Service type" as either Elasticsearch or OpenSearch. Choose OpenSearch and give your new service a name.
+
+Use the new OpenSearch service to test your application against before performing an upgrade. We recommend upgrading any dependencies at the same time.
+
+Fork an existing Elasticsearch to Elasticsearch
+-----------------------------------------------
+
+The database fork acts as a clone of your existing Elasticsearch service, but allows you to test that an upgrade will run smoothly.
+
+1. In the service overview of the web console, choose the "New database fork" button to create a new service from a backup.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:30]: "New database fork" could be bolded to keep consistency for the UI references

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+2. Check that the "Service type" field is set to Elasticsearch and give your new service a name.
+
+You can use this service to test the upgrade to OpenSearch (instructions in the next section).
+
+Upgrade from Elasticsearch to OpenSearch
+----------------------------------------
+
+When you are ready to perform the upgrade, visit the service overview page and choose the button labelled "Upgrade Elasticsearch". This operation cannot be reversed.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:38]: "Upgrade Elasticsearch" could be bolded to keep consistency for the UI references

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+... that's all. When the operation completes, your Aiven for OpenSearch service should continue to operate much as your Elasticsearch service did.
```
Comment (diawarr) [inline docs/products/opensearch/howto/upgrade-to-opensearch.rst:40]: I'd remove the "... that's all" as unnecessary

File: docs/products/opensearch/reference/plugins.rst [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+* `Open Distro for Elasticsearch Alerting plugin <https://opendistro.github.io/for-elasticsearch/features/alerting.html>`_
+
+Additional plugins
+------------------
+
+To enable additional plugins in an OpenSearch service:
+
+1. Select your service in the Aiven web console.
+2. On the service overview page, scroll down to Additional plugins
+   switch on the feature.
```
Comment (diawarr) [inline docs/products/opensearch/reference/plugins.rst:33]: Missing "and" >> and switch on the feature.

File: docs/products/opensearch/concepts/opensearch_backups.rst (hunk 1 above)
Comment (lornajane) [reply docs/products/opensearch/concepts/opensearch_backups.rst:20]: Will do

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst (hunk 3 above)
Comment (lornajane) [reply docs/products/opensearch/howto/upgrade-to-opensearch.rst:4]: Sounds much nicer, thanks

File: docs/products/opensearch/howto/upgrade-to-opensearch.rst (hunk 10 above)
Comment (lornajane) [reply docs/products/opensearch/howto/upgrade-to-opensearch.rst:40]: it just seemed like an anticlimax! But you are right of course

**Review by diawarr — APPROVED** [review]
Looks good ;)

**Pull request merged**